src/System.Text.Primitives/System/Text/FormattingData.cs

```diff
@@ -142,6 +503,9 @@ public enum Symbol : ushort
             PlusSign = 14,          
             NaN = 15,
             Exponent = 16,
+            ExponentSecondary = 17,
```

Comment:
I was thinking that I'd need a separate symbol for when there was more than one way to represent a symbol ('e', 'E'), but now I just realized that was dumb and they should both point to 16. Changing this haha.